addons/bestja_volunteer/views/report_account_deletion_wizard.xml

```diff
@@ -0,0 +1,37 @@
+<openerp>
+    <data>
+        <!-- Adding "Usuń konto" in Preferences -->
+        <record model="ir.ui.view" id="view_users_form_simple_modif2">
+            <field name="name">bestja.volunteer.delete_account</field>
+            <field name="model">res.users</field>
+            <field name="inherit_id" ref="base.view_users_form_simple_modif"/>
+            <field name="arch" type="xml">
+                <button name='preference_change_password' position="after">
+                    <button name="%(create_account_deletion_wizard)d" type="action" string="Usuń konto" class="oe_link"/>
+                 </button>
```

Comment:
You have position="after" in this button, that means that what's inside the tags it after the original button. You can use xpath instead.